app/Module.java

```diff
@@ -10,22 +14,58 @@
 import com.commercetools.sunrise.common.template.i18n.I18nResolver;
 import com.commercetools.sunrise.framework.MultiControllerComponentResolver;
 import com.commercetools.sunrise.framework.MultiControllerComponentResolverBuilder;
+import com.commercetools.sunrise.payments.PaymentConfiguration;
+import com.commercetools.sunrise.productcatalog.productoverview.search.facetedsearch.FacetedSearchConfigList;
+import com.commercetools.sunrise.productcatalog.productoverview.search.facetedsearch.FacetedSearchConfigListProvider;
+import com.commercetools.sunrise.productcatalog.productoverview.search.productsperpage.ProductsPerPageConfig;
+import com.commercetools.sunrise.productcatalog.productoverview.search.productsperpage.ProductsPerPageConfigProvider;
+import com.commercetools.sunrise.productcatalog.productoverview.search.sort.SortConfig;
+import com.commercetools.sunrise.productcatalog.productoverview.search.sort.SortConfigProvider;
 import com.commercetools.sunrise.shoppingcart.MiniCartControllerComponent;
 import com.commercetools.sunrise.shoppingcart.common.CheckoutCommonComponent;
 import com.commercetools.sunrise.shoppingcart.common.CheckoutStepWidgetComponent;
 import com.google.inject.AbstractModule;
 import com.google.inject.Provides;
+import com.google.inject.name.Names;
+import io.sphere.sdk.categories.CategoryTree;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.payments.PaymentMethodInfoBuilder;
 
+import javax.inject.Named;
 import javax.inject.Singleton;
+import java.util.Locale;
+
+import static java.util.Collections.singletonList;
 
 public class Module extends AbstractModule {
 
     @Override
     protected void configure() {
+        bind(SphereClient.class).toProvider(RequestScopedSphereClientProvider.class).in(RequestScoped.class);
```

Comment:
Basically the rule of thumb to decide whether to put a binding in this module or in SunriseModule is how likely is that the developer would like to override this binding. In SunriseModule is not possible to override without having to create your own SunriseModule with the desired changes, whereas Module is supposed to be in the project itself so you just have to modify the code.
Therefore I've put in SunriseModule the `global` SphereClient because it's a special client that people shouldn't need to override. But this one, is very likely that they'll want to add some metrics or wrap it for some other purpose.